nosqlbench/http-jsonapi-search-filter-sort.md

```diff
@@ -0,0 +1,79 @@
+# JSON API CRUD Basic
+
+## Description
+
+The JSON API CRUD Advanced workflow targets Stargate's JSON API using generated JSON documents.
```

Comment:
update description (workflow name, anything else that's different here like what sort options we are testing)